postman/campus-kid.postman_collection.json

```
@@ -658,7 +658,7 @@
 									}
 								},
 								"url": {
-									"raw": "http://127.0.0.1:4005/api/course",
+									"raw": "http://127.0.0.1:4005/api/course/save",
```

```suggestion
									"raw": "http://127.0.0.1:4005/api/course",
```
The `/save` in the URL is redundant. REST provides methods that allow us to choose the action we want to perform on the database.

For example, the POST method tells us that an endpoint will create a new element in the database. Then, when we execute a POST method to the endpoint `api/course/` we know we will create a new course in the database.

Please use this concept to model your API.